appeals/web/src/server/lib/mappers/appeal/submappers/case-officer.mapper.js

```diff
@@ -13,33 +15,17 @@ import { surnameFirstToFullName } from '#lib/person-name-formatter.js';
 			? `<ul class="govuk-list"><li>${surnameFirstToFullName(caseOfficerUser?.name)}</li><li>${
 					caseOfficerUser?.email
 			  }</li></ul>`
-			: '<p class="govuk-body">A case officer is assigned but their user account was not found';
+			: '<p class="govuk-body">A case officer is assigned but their user account was not found</p>';
```

Comment:
I don't think this branch can ever be reached anyway.

Reply:
IT's reached if someone is assigned to the case but their email is removed from AzureAd. That being said I think they do a soft delete first (so rename the user to DELETED_user123) which should allow us to monitor for them in MyPINS

Reply:
I think the logic is wrong though, there's an if statement:

`if (caseOfficerUser && !skipAssignedUsersData) {`

then ternary checking caseOfficerUser: `caseOfficerRowValue = caseOfficerUser ?...`

So that ternary will only ever be true because of the if statement its within.